packages/sitecore-jss-react/src/components/Placeholder.test.tsx

```diff
@@ -677,18 +709,46 @@ it('should render custom HiddenRendering when rendering is hidden', () => {
   );
 
   const renderedComponent = mount(
-    <Placeholder
-      name={phKey}
-      rendering={route}
-      componentFactory={componentFactory}
-      hiddenRenderingComponent={CustomHiddenRendering}
-    />
+    <SitecoreContext componentFactory={componentFactory}>
+      <Placeholder
+        name={phKey}
+        rendering={route}
+        hiddenRenderingComponent={CustomHiddenRendering}
+      />
+    </SitecoreContext>
   );
   expect(renderedComponent.find('.hidden-rendering').length).to.equal(1);
   expect(renderedComponent.find(HiddenRendering).length).to.equal(1);
   expect(renderedComponent.find('p').props().children).to.equal('Hidden Rendering');
 });
 
+const testDataWithoutEE = [
```

Comment:
```suggestion
const testDataWithMetadata = [
```

Reply:
And something is definitely wrong here. You are relying on non-editing layout service data, but your unit test should cover editing scenario and inside of unit test you are referencing _layoutDataWithMetadata_ that is confusing